autoload/airline/extensions.vim

```diff
@@ -311,6 +311,11 @@ function! airline#extensions#load()
     call add(s:loaded_ext, 'ale')
   endif
 
+  if (get(g:, 'airline#extensions#lsp#enabled', 1) && exists(':LspDeclaration'))
```

Comment:
I thought it was better to specify `LspStatus` or `g:lsp_loaded` than `LspDeclaration`. This is just a question, why did you specify `LspDeclaration`?

Reply:
I'm not sure it's important. vim-lsp exports both functions.